src/components/page/elements/ScriptElement.js

```diff
@@ -1,41 +1,45 @@
-import shajs from 'sha.js';
+import { getSHA256 } from '../../../lib/helpers'
 
 export default function ScriptElement(props) {
-    handleScriptContent(props.content);
-    handleScriptSrc(props.element.src);
-    return (null);
+  handleScriptContent(props.content)
+  handleScriptSrc(props.element.src)
+  return null
 }
 
 function getElementByHash(value) {
-    const scriptContentHash = shajs('sha256').update(value).digest('hex');
-    const doesAlreadyExist = document.getElementById(scriptContentHash);
-    return {hash: scriptContentHash, exists: doesAlreadyExist}
+  const scriptContentHash = getSHA256(value)
+  const doesAlreadyExist = document.getElementById(scriptContentHash)
+  return { hash: scriptContentHash, exists: doesAlreadyExist }
 }
 
 function handleScriptContent(content) {
```

Comment:
A function should return as soon as it knows it can.
Here, negating assertions would make the code more readable
```javascript
if (!content) return

// same within the forEach block
if (itm === '') return
const element = ..

if (element.exists) return

...


```

Reply:
👍